blockchain/src/history_store/mmr_store.rs

```diff
@@ -150,4 +150,8 @@ impl<'a, 'env> Store<HistoryTreeHash> for MMRStore<'a, 'env> {
     fn len(&self) -> usize {
         self.size
     }
+
+    fn is_empty(&self) -> bool {
```

Comment:
Is this part of some clippy warning? (Same for all of the other places where you added the `is_empty()` function)

Reply:
Yup, the warning is about having a `len` method without an `is_empty` as documented [here](https://rust-lang.github.io/rust-clippy/master/index.html#len_without_is_empty).